service/src/main/java/dk/dbc/search/solrdocstore/AddResourceRequest.java

```diff
@@ -0,0 +1,15 @@
+package dk.dbc.search.solrdocstore;
+
+public class AddResourceRequest extends BibliographicResource {
+
+    public AddResourceRequest(){}
+
+    public AddResourceRequest(int agencyId, String field, String bibliographicRecordId, boolean value){
+        super(agencyId, field, bibliographicRecordId, value);
+    }
+
+    BibliographicResource asBibliographicResource() {
+        return new BibliographicResource(this.getAgencyId(), this.getField(),
+                this.getBibliographicRecordId(), this.getValue());
+    }
+}
```

Comment:
Hvad er formålet med at pakke BibliographicResource in i en ny klasse. Alle steder bruges objekterne som om de er et BibliographicResource.